compiler/luci-eval-driver/src/EvalDriver.cpp

```diff
@@ -98,8 +99,9 @@ int entry(int argc, char **argv)
     return EXIT_FAILURE;
   }
 
+  auto memory_manager = std::make_unique<luci_interpreter::SimpleMManager>();
```

Comment:
I think you can just create it on stack here.
like
```suggestion
luci_interpreter::SimpleMManager memory_manager;
```